app/vite.config.js

```diff
@@ -6,10 +6,35 @@ import { defineConfig } from "vite";
 import { VitePWA } from "vite-plugin-pwa";
 import { viteStaticCopy } from "vite-plugin-static-copy";
 
-// import path from "path" causes eslint to crash for some reason
+// import from "path" and "fs" causes eslint to crash for some reason
 const path = require("path");
+const { readFileSync, writeFileSync } = require("fs");
 
-export default defineConfig({
+function prodScriptPlugin(command) {
```

Comment:
```suggestion
/**
 *
 * @param {"build" | "serve"} command
 * @returns {import("vite").PluginOption}
 */
function prodScriptPlugin(command) {
```

for autocomplete and typechecking